packages/logos-docusaurus-theme/src/client/theme/DocSidebarItem/Category/index.tsx

```diff
@@ -168,7 +168,11 @@ export default function DocSidebarItemCategory({
           <Typography variant="body2" color="primary">
             {label}
           </Typography>
-          {collapsed ? <ArrowUpIcon /> : <ArrowDownIcon />}
+          {collapsed ? (
+            <ArrowUpIcon className="collapse-arrow" />
+          ) : (
+            <ArrowDownIcon />
+          )}
         </Link>
```

Comment:
Sure, let me change it in that way.

I thought it was better because I don't have to add a CSS class for this and use the`collapsed` state as it is.

Can you let me know why using a single SVG and adjusting the orientation with CSS is better? @jeangovil